docs/index/bloomfilter-index-guide.md

```diff
@@ -109,40 +110,40 @@ User can create BloomFilter DataMap using the Create DataMap DDL:
 ## Loading Data
 When loading data to main table, BloomFilter files will be generated for all the
 index_columns given in DMProperties which contains the blockletId and a BloomFilter for each index column.
-These index files will be written inside a folder named with DataMap name
+These index files will be written inside a folder named with Index name
```

Comment:
Please change the statement in Line no.112. `When loading data to main table, BloomFilter files will be generated for all the index_columns provided in the CREATE statement which contains the blockletId`

Reply:
Ok